frameworks/elastic/src/main/dist/elasticsearch.yml

```diff
@@ -21,3 +21,7 @@ node.data: {{DATA_ENABLED}}
 node.ingest: {{INGEST_ENABLED}}
 metrics.statsd.host: {{STATSD_UDP_HOST}}
 metrics.statsd.port: {{STATSD_UDP_PORT}}
+gateway.expected_master_nodes: 3
```

Comment:
https://discuss.elastic.co/t/gateway-expected-nodes-and-full-cluster-restart/97151/2?u=loren

I changed the template to only apply to masters.